docs/file-types.md

```diff
@@ -15,40 +17,43 @@ GIF (image/gif)
 BMP (image/bmp)
 ```
 
-These are enabled by having the following in your `config.php`,
+These are enabled by inclusion of the following values in `config.php`'s `enabledPreviewProviders` array:
 
 ```php
-'enabledPreviewProviders' =>
-array (
-  'OC\\Preview\\Image',
-),
+  'OC\Preview\Image',
 ```
 
-## HEIC and TIFF
+... or...
+
+```php
+  'OC\Preview\PNG',
+  'OC\Preview\JPEG',
+  'OC\Preview\GIF',
+  'OC\Preview\BMP',
+  'OC\Preview\XBitmap',
+```
```

Comment:
I added this because my config actually lists the file types individually. Not sure if that's a byproduct of being on a newer/older installation than you, but it seemed worth including.

Reply:
Reverted this one since it's [pointless](https://github.com/nextcloud/server/blob/master/lib/private/Preview/JPEG.php) and (academically) would hurt performance. Thanks for the PR!